### ISSUE TITLE
publishToConfluence throws Error when xml with special content is included in adoc

### ISSUE THREAD

**Issue by JBengler69** [docToolchain/docToolchain#1265]
I receive an error when I add a source block to an asciidoc file that contains a CDATA section holding another xml document during the publishToConfluence task. generateHTML runs fine for the same asciidoc

**To Reproduce**
1. Add a source block containing an XML document like:
```xml
.Unzipped Payload
====
[source, xml]
<?xml version="1.0" encoding="UTF-8"?>
<Resultset>
	<EXPORTDATA>
		<EXDA_ID>50849357</EXDA_ID>
		<EXDA_CREATIONDATE>2022-10-07T16:35:33</EXDA_CREATIONDATE>
		<EXDA_PROCESSINGDATE>2022-10-07T16:35:33</EXDA_PROCESSINGDATE>
		<EXDA_GUID>f4f73eb9-34e1-4d76-9ad4-43a8f6dbcf65</EXDA_GUID>
		<EXDA_DATA>
			<![CDATA[&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;Resultset xmlns:xsi=&quot;http://www.w3.org/2001/XMLSchema-instance&quot;&gt;
		... omitted for better readability
&lt;/Resultset&gt;]]>
		</EXDA_DATA>
		<EXDA_PRST_INDEX>10</EXDA_PRST_INDEX>
		<EXDA_CL_INDEX>002</EXDA_CL_INDEX>
		<EXDA_DESCRIPTOR1>DARMAULA</EXDA_DESCRIPTOR1>
		<EXDA_DESCRIPTOR2>DAK8441640       I01DA0101</EXDA_DESCRIPTOR2>
		<EXDA_DESCRIPTOR3>20220804F349</EXDA_DESCRIPTOR3>
		<EXDA_DESCRIPTOR4>361</EXDA_DESCRIPTOR4>
		<EXDA_DESCRIPTOR5>word_date1 20220804133700</EXDA_DESCRIPTOR5>
		<EXDA_DESCRIPTOR6/>
		<EXDA_BUSINESSTYPE>RMORD</EXDA_BUSINESSTYPE>
		<EXDA_SD_NAME>RMORD</EXDA_SD_NAME>
		<EXDA_SD_VERSION>2.3</EXDA_SD_VERSION>
		<EXDA_TRHE_ID>6636702</EXDA_TRHE_ID>
		<EXDA_TRPO_ID>1</EXDA_TRPO_ID>
		<EXDA_PRIO>4</EXDA_PRIO>
		<EXDA_TRPA_SENDER>AU.002</EXDA_TRPA_SENDER>
		<EXDA_TRPA_RECEIVER>2.15</EXDA_TRPA_RECEIVER>
	</EXPORTDATA>
</Resultset>
====
```

2. run dtcw publishToConfluence
3. See error:
	something went wrong - got an http response code 400:
	Error parsing xhtml: Unexpected close tag </Resultset>; expected </ac:plain-text-body>.
	 at [row,col {unknown-source}]: [311,11]
	please check the ancestorId in your config file

	> Task :publishToConfluence FAILED

	FAILURE: Build failed with an exception.

	* Where:
	Script 'C:\Users\bj20138\.doctoolchain\docToolchain-3.0.2\scripts\publishToConfluence.gradle' line: 30

	* What went wrong:
	Execution failed for task ':publishToConfluence'.
	> java.lang.Exception: Parent does not exist

**Expected behavior**
XML file should be displayed in an example block on the published confluence page

**Configuration**
 - docToolchain version 3.0.2
 - OS: Windows 10
 - Browser Chrome
 - Relevant SDK Versions openjdk version "11.0.4-redhat" 2019-07-16 LTS

**Comment by PacoVK** [comment]
Thanks for reporting this issue. I included the fix in the confluence api upgrade, which will be released soon 🙌

**Comment by PacoVK** [comment]
@JBengler69  could you please check if the Issue has been solved from v3.1.1?

**Comment by JBengler69** [comment]
@PacoVK  Thank you, I just tried and it worked well!



Just one comment (I think it doesn't concern you): when I installed v3.1.1 by creating a new project folder, renaming the existing .doctoolchain folder in my user directory (to hide the existing installation), downloading the dtcw.bat file and calling "dtcw install doctoolchain" it installs the 3.1.1 in a new .doctoolchain folder, but when I try to run it I receive an error:

"FAILURE: Build failed with an exception.

* What went wrong:
Included build 'C:\Users\bj20138\.doctoolchain\docToolchain-3.1.1\core' does not exist."

And indeed, the core subfolder did not exist...

So I downloaded the source from github and copied the core folder to C:\Users\bj20138\.doctoolchain\docToolchain-3.1.1\ and everything worked fine.
